### ISSUE TITLE
Allow 'copyAll' for all sorting modes

### ISSUE THREAD

**Issue by fritzmg** [contao/core#7499]
It seems the "copyAll" function is only available for DCAs with a sorting mode that is not [0,1,2]. Which means that, if you want to be able to copy multiple entries, your data structure needs a parent.

Is there any reason for this restriction? Why is multi-copy not available for all sorting modes?

**Comment by fritzmg** [comment]
Ah, well yes, I think there should be ;)

I noticed it recently, when I was creating shipping method entries in Isotope. I wanted to copy a lot of existing entries and then change their country settings and prices for instance. Had to do it directly via SQL queries instead.

I also tried to quickly hack in the multi copy function, but I noticed there is a lot more to it than simply allowing the copyAll function to work with mode [0,1,2] tables. Even extensions themselves (like Isotope) would need to make some changes. For instance the `checkPermissions` functions need to add an additional switch case for `copyAll` and insert the (allowed) IDs into `$session['CURRENT']['IDS']`, which is currently not done, neither in the core nor in `Isotope\Backend\Shipping\Callback` for example.
